Makefile

```diff
@@ -46,6 +46,7 @@ appstore:
 	--exclude=phpunit*xml \
 	--exclude=tests \
 	--exclude=vendor/bin \
+	--exclude=vendor/yubico/u2flib-server/examples \
```

Comment:
I'd rather prefer to remove line 27. The stuff is already checked in in git anyways